Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -1,19 +1,164 @@
-.. |author| replace:: *TDB*
+.. |author| replace:: *Yijung Kang*
 .. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
-.. |contributors| replace:: *none*
+.. |contributors| replace:: *Alysha Shugart, Karla Aubel*
 
 .. _AT-Shutdown-Shutdown:
 
 #######################
 Shutdown
 #######################
 
-Under construction.
+
+.. _Shutdown-the-Telescope-Overview:
+
+Overview
+========
+
+At the end of the night, the telescope, LATISS, CSCs compoents, and Dome should be in shutdown and/or standby status.
+The parking position is for the AuxTel is El = 80 deg, Az = 0.0 deg, rot = 0.0 deg. 
+For the dome, it is at az = 285.0 deg with closed dome slit shutter.
+All ATTCS and LATISS CSCs for AuxTel should be set to standby.  
+
+
+.. _Shutdown-the-Telescope-Prerequisites:
+
+Prerequisites
+=============
+
+Stopping Scheduler
+------------------
+
+The Scheduler automatically adds new targets to the queue until the start of nautical morning twilight.
+If there are no targets, the Scheduler will stop adding new ones.
+If you need to stop the Scheduler manually, you can use the ``auxtel/scheduler/stop.py`` script from ScriptQueue without any additional configurations.
+
+See more detailed descriptions on `Scheduler <https://obs-ops.lsst.io/Nighttime-Operations/Scheduler/night-time-operations/index.html>`__.
+
+
+.. _Shutdown-the-Telescope-Post-Conditions:
+
+Post-Condition
+==============
+
+- **Status of Telescope and Dome** 
+.. image:: ./_static/ATDome_park.png
+    :name: Auxiliary Telescope and Dome at the parking position
+
+- **Status of Mirrors and Mirror Covers**
+.. image:: ./_static/AT_LightPath.png
+   :scale: 50 %
+
+- **Status of CSC components of ATTCS and LATISS** 
+.. image:: ./_static/CSC.png
+
+
+
+.. _Park-the-Telescope-Procedure-Steps:
+
+Procedure Steps
+===============
+
+Shutdown AuxTel
+---------------
+
+After stopping scheduler, you can shutdown AuxTel using ``axutel/shutdown.py`` script from ATQueue. 
+``shutdown.py`` script does not any configurations as below. 
+
```

Comment:
you mean does not use any configuration? the frase is a bit confusing.

Reply:
Sorry, my bad. I revised 65 lines to 
"``shutdown.py`` script does not have any configurations as below. "